docs/grammar/SolidityLexer.g4

```diff
@@ -288,6 +288,8 @@ YulPeriod: '.';
 YulComma: ',';
 YulArrow: '->';
 
+YulHex: 'hex';
+
```

Comment:
I saw this in the solidity lexer part... not entirely sure if this is actually needed... and it might cause problems, in case we intend to *allow* ``let hex := 42``...
And if we *disallow* ``let hex := 42``, then this change would strictly speaking even be breaking, wouldn't it :-)?

Reply:
If it even was valid before that is...